capa/ida/plugin/README.md

```diff
@@ -0,0 +1,100 @@
+# capa explorer
+
+capa explorer is an IDA Pro plugin that integrates the FLARE team's open-source framework 
+[capa](https://www.fireeye.com/blog/threat-research/2020/07/capa-automatically-identify-malware-capabilities.html) with IDA. You can
+use capa explorer to quickly identify and navigate to interesting areas of a program and dissect capa rule matches at
+the assembly level.
+
+For example, we run capa explorer against a suspicious executable loaded into IDA and see that capa detected a rule match
+for `self delete via COMSPEC environment variable`:
+
+![](../../../doc/img/ida_plugin_example_1.png)
+
+We can use capa explorer to navigate the IDA Disassembly view directly to the suspect function and get an assembly-level breakdown
+of why capa matched `self delete via COMSPEC environment variable` for this particular function.
+
+![](../../../doc/img/ida_plugin_example_2.png)
+
+## Features
+
+![](../../../doc/img/ida_plugin_intro.gif)
+
+* Display capa results in an interactive tree view of rule matches and their locations in the current database
+* Export results as formatted JSON by navigating to `File > Export results...`
+* Remember a user's capa rules directory for future runs; change capa rules directory by navigating to `Rules > Change rules directory...`
+* Search for keywords or phrases found in the `Rule Information`, `Address`, or `Details` columns
+* Display rule source content when a user hovers their cursor over a rule match
+* Double-click `Address` column to view associated feature in the IDA Disassembly view
+* Limit tree view results to the function currently displayed in the IDA Disassembly view; update results as a user navigates to different functions
+* Sort results by column
+* Reset tree view and IDA Disassembly view highlighting by clicking `Reset`
+* Automatically re-analyze database when user performs a program rebase
+* Automatically update results when IDA is used to rename a function
+* Select one or more checkboxes to highlight the associated addresses in the IDA Disassembly view
+* Right-click a function match to rename it; the new function name is propagated to the current IDA database
+* Right-click to copy a result by column or by row
```

Comment:
this list is rather long. i think its a reasonable compromise between authoring a bunch of text that few people will read closely and not providing enough detail. this is enough to get started (and excited).

to emphasize the most important/interesting elements, put them first.